packages/harpguru-core/src/utils/get-output-range/get-output-range.ts

```diff
@@ -2,6 +2,8 @@ export const getOutputRange = (
   index: number,
   rangeLength: number
 ): ReadonlyArray<number> => {
+  'worklet'
+  // TODO: I think this can be moved closer to it's only dependent
```

Comment:
It's dependant is in the package root `hooks` folder so this can't really move closer unless it's incorporated in to the file. That could be done but that file's destiny is also in question.

Reply:
TODO removed as stated here: https://github.com/js-jslog/harpguru-lift-shift-sdk-47/pull/7#discussion_r1059788892